webroot/src/models/PrivilegePurchaseOption/PrivilegePurchaseOption.model.spec.ts

```diff
@@ -0,0 +1,121 @@
+//
+//  PrivilegePurchaseOption.model.spec.ts
+//  CompactConnect
+//
+//  Created by InspiringApps on 7/8/2024.
+//
+
+import { expect } from 'chai';
+import { FeeTypes } from '@/app.config';
+import {
+    PrivilegePurchaseOption,
+    PrivilegePurchaseOptionSerializer
+} from '@models/PrivilegePurchaseOption/PrivilegePurchaseOption.model';
+import { State } from '@models/State/State.model';
+import i18n from '@/i18n';
+
+describe('PrivilegePurchaseOption model', () => {
+    before(() => {
+        const { tm: $tm } = i18n.global;
+
+        (window as any).Vue = {
+            config: {
+                globalProperties: {
+                    $tm,
+                }
+            }
+        };
+        i18n.global.locale = 'en';
+    });
+    it('should create a PrivilegePurchaseOption with expected defaults', () => {
+        const privilegePurchaseOption = new PrivilegePurchaseOption();
+
+        // Test field values
+        expect(privilegePurchaseOption).to.be.an.instanceof(PrivilegePurchaseOption);
+        expect(privilegePurchaseOption.jurisdiction).to.be.an.instanceof(State);
```

Comment:
any particular state? should we test that it's an "empty" state?

generally, why would a privilege purchase option ever be instantiated without the requisite variables (jurisdiction, compact, fee, etc)?

Reply:
Generally, I don't think it would be but this test confirms that the defaults are set correctly in case it does.

Ill add a test to confirm the empty state.